lib/open_mic.rb

```diff
@@ -0,0 +1,18 @@
+class OpenMic
+
+  attr_reader :location, :date, :performers
+
+  def initialize(location: "Comedy Works", date: "11-20-18")
```

Comment:
Couple things here-- the way you have these written, you have these as default values-- that means that every new instance of an open mic that you create will instantiate with a location of comedy works and a date of 11-20-18.  It's not necessarily _bad_, because you can technically always override those values.. but it also doesn't really make sense.  If you think about it in terms of what makes sense as a human, do you expect most open mic sessions every to be specifically located at comedy works, on that specific date?  (that's ultimately what we're trying to say when we provide default values)  There's also nothing in the interaction pattern that suggestions that we should be using default values here, so for those two reasons, I'd expect to see this information passed in without them.

Also note, the interaction pattern shows this: `open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"}) ` 

If you look carefully, you'll see that `OpenMic.new` technically is only taking _one_ argument-- a single hash (although it's a single hash with two key/value pairs). You have your OpenMic set up to take two separate arguments, so your class doesn't quite match what the interaction pattern is asking for.  You'd instead want do this similarly to what we talked about in the hash class at the very end (take a look at that Unicorn class you requested via Slack).  

In any case, no worries-- but something to watch out for and be aware of next time 😄

Reply:
I will say-- I have a sneaking suspicion that although you wrote your open mic test, that you might not have actually been running it, or at least not consistently-- because the way you wrote this class requires two arguments, but the way you initialize it in your test setup _does_ match the interaction pattern, which only provides one argument (the single hash), so I'd expect all of your open mic tests to fail due to a wrong number of arguments error. 🤷‍♀